web-common/src/features/dashboards/dimension-table/dimension-table-utils.ts

```diff
@@ -17,7 +18,7 @@ import type { VirtualizedTableConfig } from "@rilldata/web-common/components/vir
 import type { SvelteComponent } from "svelte";
 import { getDimensionColumn } from "../dashboard-utils";
 import type { DimensionTableRow } from "./dimension-table-types";
-import { getFilterForDimension } from "../selectors";
+import { getFiltersForOtherDimensions } from "../selectors";
```

Comment:
Note: other than changing the name of this function, there are no substantive changes in this file. Everything else is just type guards and the like, but the diff looks messy because Prettier changed the formatting a lot